internal/model/notif_ntfy.go

```diff
@@ -0,0 +1,34 @@
+package model
+
+import (
+	"time"
+
+	"github.com/crazy-max/diun/v4/pkg/utl"
+)
+
+// NotifNtfy holds ntfy notification configuration details
+type NotifNtfy struct {
+	Endpoint      string         `yaml:"endpoint,omitempty" json:"endpoint,omitempty" validate:"required"`
+	Topic         string         `yaml:"topic,omitempty" json:"topic,omitempty" validate:"required"`
+	Priority      string         `yaml:"priority,omitempty" json:"priority,omitempty" validate:"omitempty,min=0"`
+	Timeout       *time.Duration `yaml:"timeout,omitempty" json:"timeout,omitempty" validate:"required"`
+	TemplateTitle string         `yaml:"templateTitle,omitempty" json:"templateTitle,omitempty" validate:"required"`
+	TemplateBody  string         `yaml:"templateBody,omitempty" json:"templateBody,omitempty" validate:"required"`
+	Emoji         string         `yaml:"emoji,omitempty" json:"emoji,omitempty" validate:"required"`
```

Comment:
Like my previous comment it should be named `Tags` and be a slice of string following the API spec.

It should also be placed before `Timeout` for consistency with other notifiers.